urlpattern/resources/urlpatterntestdata.json

```diff
@@ -1145,6 +1145,14 @@
   {
     "pattern": [{ "protocol": "http", "port": "80 " }],
     "inputs": [{ "protocol": "http", "port": "80" }],
+    "exactly_empty_components": ["port"],
+    "expected_match": {
+      "protocol": { "input": "http", "groups": {} }
+    }
+  },
+  {
+    "pattern": [{ "protocol": "http", "port": "100000" }],
+    "inputs": [{ "protocol": "http", "port": "100000" }],
```

Comment:
I don't quite follow what the intention of this test case is. Could you help me to understand?

Reply:
Definitely. It tests the maximum valid port number. If it is greater than 65k, it should throw (according to url spec)